projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-container *ngIf="query">
+    <p>
+        <br><a class="govuk-link" href="javascript:void(0)" (click)="onBack()">Back to queries</a>
+    </p>
+    <div class="query_details_caption">Query details</div>
```

Comment:
@DavidJayakumar - `Query details` needs to run through `rpxTranslate` pipe for Welsh language support

Reply:
As per Dan's comment